docs/source/indexes/cagra.rst

```diff
@@ -98,53 +98,143 @@ Tuning Considerations
 The 3 hyper-parameters that are most often tuned are `graph_degree`, `intermediate_graph_degree`, and `itopk_size`.
 
 Memory footprint
-----------------
+================
 
-CAGRA builds a graph that ultimately ends up on the host while it needs to keep the original dataset around (can be on host or device).
+CAGRA builds a nearest-neighbor graph(neighbor IDs) that ultimately ends up on the host while it needs to keep the original dataset(raw vectors) around (can be on host or device)
```

Comment:
Small improvement here- can we maybe mention briefly that "keeping the dataset around" can be optional when immediately converting CAGRA to a CPU-based format like HNSW for search?